packages/suite/src/views/dashboard/components/AssetsView/components/AssetCoinLogo.tsx

```diff
@@ -28,12 +28,18 @@ export const AssetCoinLogo = ({ symbol, assetsFiatBalances, index }: CoinLogoPro
 
     return (
         <LogoWrapper>
-            <AssetShareIndicator
-                symbol={symbol}
-                size={24}
-                percentageShare={assetPercentage}
-                index={index}
-            />
+            <Tooltip
+                content={`${(assetPercentage ?? 0).toFixed(2)} %`}
+                cursor="pointer"
+                delay={1000}
```

Comment:
Nice touch!

nit: delay={[600, 0]} for all [label tooltips](https://github.com/trezor/trezor-suite/blob/23667022e37223dd6c446ce2e3c4af9fc91b5422/packages/components/src/components/buttons/IconButton/IconButton.tsx#L59) (I should create a constant!) and probably no delay here, actually.

Reply:
https://github.com/trezor/trezor-suite/pull/10973